internal/pkg/utils/testproject/project.go

```diff
@@ -314,6 +320,44 @@ func (p *Project) createBranches(branches []*fixtures.BranchState) error {
 	return nil
 }
 
+func (p *Project) createBucketsTables(buckets []*fixtures.Bucket) error {
+	ctx, cancelFn := context.WithCancel(context.Background())
+	defer cancelFn()
+
+	// Create buckets
+	grp := client.NewWaitGroup(ctx, p.storageApiClient)
+	for _, b := range buckets {
+		grp.Send(storageapi.CreateBucketRequest(&storageapi.Bucket{
+			Name:        b.Name,
+			Stage:       b.Stage,
+			Description: b.Description,
+		}))
+	}
+	if err := grp.Wait(); err != nil {
+		return err
+	}
+
+	// Create tables
+	grp = client.NewWaitGroup(ctx, p.storageApiClient)
+	for _, b := range buckets {
```

Comment:
Hned ako sa vytvori bucket sa daju vytvarat v nom tabulky, nie?
Neda sa to napojit vyssie cez `WithOnSuccess`?

Reply:
Jo ok, přemýšlel jsem nad tím, ale asi to vlastně bude o něco rychlejší. 🙂

Reply:
Upraveno: https://github.com/keboola/keboola-as-code/pull/881/commits/ad2cf4259a3d27ab3e32a4760ef61914159268d5